src/components/OAuth2AppCreateRecordForm.vue

```diff
@@ -80,5 +97,16 @@ export default {
       }).then(this.showResult)
     },
   },
+
+  computed: {
```

Comment:
Why don't you just use the data property directly instead of creating a wrapper on it?

Reply:
I just copied this out of `RecordManageModal` ¯\\\_(ツ)_/¯ I think it was originally written that way because we wanted the toggle switch to be "off" by default, but the API boolean it "on" by default. The wording would have to change if we flipped it.